src/models/model.py

```diff
@@ -21,12 +21,13 @@ def __init__(self, config: DictConfig):
                 output_attentions=False,
                 output_hidden_states=False,
             )
-        else:  # default model is distilbert
+        elif (
+            self.config.model["model"] == "distilbert-base-uncased"
```

Comment:
I thought of having distilBert as the default model in case some configs are wrong.
Perhaps we can do it like 
```suggestion
            else:
              print("Using DistilBert")
              self.model = DistilBertForSequenceClassification.from_pretrained(
                  "distilbert-base-uncased",
                  num_labels=self.config.model["num_labels"],
                  )
```